include/librealsense2-gl/rs_processing_gl.h

```diff
@@ -75,6 +75,16 @@ struct glfw_binding
 */
 rs2_processing_block* rs2_gl_create_yuy_decoder(int api_version, rs2_error** error);
 
+/**
+* REWRITE!!!!
+* Creates a processing block that can efficiently convert YUY image format to RGB variants
```

Comment:
Maybe it's better to remove optimizations from this PR?

Reply:
Let's talk about this